docs/sources/administration/service-accounts/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Service accounts in Grafana'
+menuTitle: 'Service accounts'
+description: 'This contains information about service accounts in Grafana'
+aliases: [docs/sources/administration/service-accounts/_index.md]
+weight: 300
+keywords:
+  - Api keys
+  - Service accounts
+---
+
+# Service accounts in Grafana
+
+A service account is generally a regular user account, but used to run automated or compute workloads with care for specific use cases.
```

Comment:
Technical content should not include adverbs, which (most of the time) end in '-ly'. You can remove 'generally'. Also not sure what 'regular' means here. I would just simply this sentence to something like:

'A service account is a special type of user account that you can use to run automated or compute workloads.'

Reply:
@vtorosyan the service account can be used by teams to interact with grafana as well. In a sense acting like a user. But this is ofc people that are developers and know that you can do this kind of stuff.